docs/site/Community-extensions.md

```diff
@@ -0,0 +1,92 @@
+---
+lang: en
+title: 'Community extensions'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, OpenAPI, Extensions
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Community-extensions.html
+---
+
+In addition to the extensions that LoopBack provides and maintains, there are a
+number of extensions created by the open source community.
+
+To create a LoopBack extension, see
+[Extending LoopBack 4 documentation page](https://loopback.io/doc/en/lb4/Extending-LoopBack-4.html)
+for details. It is recommended to use the
+[extension generator](https://loopback.io/doc/en/lb4/Extension-generator.html)
+to scaffold a new extension.
+
+The following table lists some of the community connectors. See
+[npmjs.org](https://www.npmjs.com/search?q=loopback4%20extension) for a complete
+list.
+
+{% include warning.html content="The extensions listed here are not supported by the LoopBack team; they are maintained by the LoopBack community and are listed here for convenience."%}
+
+## General
+
+<table>
+  <thead>
+    <tr>
+      <th width="160">Extension</th>
+      <th width="280">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><a href="https://www.npmjs.com/package/loopback4-soft-delete">loopback4-soft-delete</a></td>
+      <td>Used for soft-deletes in a multi-tenant application</td>
```

Comment:
This extension is generic not specifically for multi tenant application. Can you reword that please?

Reply:
do you have a suggestion?

Reply:
An extension for adding soft-delete feature to your models and repositories